lifechanger/src/app/services/project-card.service.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@angular/core';
+import { ProjectCardsInfo } from '../models/project-card';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ProjectCardsService {
+  GetProjectCardsInfos(): ProjectCardsInfo[] {
+    return [
+      {
+        imageAndAlt: {
```

Comment:
If you remember comment related to renaming, it's one of places where you'll need adjust naimg in case if you try to do renaming

Reply:
Thank you, dear Mentor, for the remark, I repaired:)